packages/e2e-test-utils-playwright/src/editor/site-editor.ts

```diff
@@ -24,6 +24,10 @@ export async function saveSiteEditorEntities( this: Editor ) {
 		.getByRole( 'button', { name: 'Save', exact: true } )
 		.click();
 
-	// Wait for the saved status.
-	await editorTopBar.getByRole( 'button', { name: 'Saved' } ).waitFor();
+	// A role selector cannot be used here because it needs to check that the `is-busy` class is not present.
+	await this.page
+		.locator( '[aria-label="Editor top bar"] [aria-label="Saved"].is-busy' )
```

Comment:
Nice find. I think it would be good to address this on an implementation level. The button should probably say `Saving`, not `Saved` as long as the request is pending (`.is-busy` ATM). cc @kevin940726

Reply:
Yes, I totally forgot to TODO comment this. Thanks!

[Relevant component file](https://github.com/WordPress/gutenberg/blob/trunk/packages/edit-site/src/components/save-button/index.js)